### ISSUE TITLE
Clippy 0.0.133 doesn't build on nightly rust.

### ISSUE THREAD

**Issue by o01eg** [Manishearth/rust-clippy#1767]
I got error when trying to install clippy:

```
error: no method named `populate_implementations_for_trait_if_necessary` found for type `rustc::ty::TyCtxt<'a, 'tcx, 'tcx>` in the current scope
   --> /home/o01eg/.cargo/registry/src/github.com-1ecc6299db9ec823/clippy_lints-0.0.133/src/utils/mod.rs:328:12
    |
328 |     cx.tcx.populate_implementations_for_trait_if_necessary(trait_id);
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `clippy_lints`.
```

It looks like it was broken in https://github.com/rust-lang/rust/pull/25323

Rust version:
```
rustc 1.19.0-dev (0ed1ec9f9 2017-05-18)
binary: rustc
commit-hash: 0ed1ec9f9eb8174273867343d01177b5ac13ff4c
commit-date: 2017-05-18
host: x86_64-unknown-linux-gnu
release: 1.19.0-dev
LLVM version: 3.9
```

**Comment by oli-obk** [comment]
I won't get to this today. But it looks like a simple renaming. If someone could whiz up a PR that would speed up the process of getting this fixed

**Comment by messense** [comment]
It's actually broken in https://github.com/rust-lang/rust/pull/41911/files#diff-c888afeccd78f53e5ea3eb04089e46ddL2333

**Comment by oli-obk** [comment]
published as 0.0.134

Thanks @messense for creating a PR!
